themes/wilson_theme_starterkit/src/scss/components/jump-nav.scss

```diff
@@ -0,0 +1,90 @@
+.jump-nav {
+  @apply sticky;
+  @apply top-[-1px];
+  @apply z-40;
+  @apply transition-colors;
+  @apply transition-shadow;
+  @apply duration-300;
+
+  &.is-sticky {
+    @apply bg-white;
+    @apply overflow-hidden;
+    @apply z-[60];
+    @apply shadow-xl;
+
+    nav {
+      @apply drop-shadow-none;
+    }
+  }
+
+  nav {
+    @apply drop-shadow-2xl;
+    @apply overflow-x-scroll;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+
+    ul {
+      @apply flex;
+      @apply p-0;
+
+      li {
+        @apply list-none;
+        @apply relative;
+
+        &::after {
+          @apply content-[""];
+          @apply absolute;
+          @apply right-0;
+          @apply top-4;
+          @apply bottom-4;
+          @apply w-[1px];
+          @apply bg-primary/50;
+        }
+
+        &:last-child {
+          &::after {
+            @apply hidden;
+          }
+        }
+      }
+    }
+  }
+
+  .jump-link {
+    @apply relative;
+    @apply inline-block;
+    @apply whitespace-nowrap;
+    @apply h-full;
+
+    &::after {
+      @apply content-[""];
+      @apply absolute;
+      @apply bottom-0;
+      @apply left-8;
+      @apply right-8;
+      @apply h-[3px];
+      @apply transition-colors;
+      @apply bg-transparent;
+    }
+
+    &:hover {
+      &::after {
+        @apply bg-primary/50;
+      }
+    }
+
+    &.is-active {
+      &::after {
+        @apply bg-primary;
+      }
+    }
+  }
+}
+
+// Offset the anchor so that the content is less likely to be obscured by a sticky jump-nav.
+.anchor-point {
```

Comment:
Could these styles have been added to the `themes/wilson_theme_starterkit/templates/paragraph/paragraph--anchor-point.html.twig` template?

Reply:
I intentionally add these styles only when the sticky jump nav is toggled on the hero. If the editor is using anchor points without the jump nav (i.e. for links inbound from other pages) then it makes more sense to align the anchor point to the top of the page.